src/app/components/reminder.js

```diff
@@ -1,42 +1,35 @@
 import axios from 'axios';
 import notify from './../base/notify';
-import { notes } from './../constants/notes';
-import { messages } from './../constants/messages';
+import { notes, messages } from './../constants/constants';
 
 export default class Reminder {
     constructor(container) {
         this.container = container;
         this.apiUrl = this.container.getAttribute('data-apiUrl');
-        this.day = this.container.querySelector('[data-reminder="day"]');
-        this.time = this.container.querySelector('[data-reminder="time"]');
-        this.setBtn = this.container.querySelector('[data-reminder="set"]');
+        this.day = this.container.querySelector('.js-selectDay');
+        this.time = this.container.querySelector('.js-selectTime');
+        this.setBtn = this.container.querySelector('.js-setReminder');
         this.setBtn.disabled = true;
-        this.SELECTEDDAY = 'Select day';
-        this.SELECTEDTIME = 'Select time';
+        this.selected_day = this.day.value;
+        this.selected_time = this.time.value;
 
-        this.day.addEventListener('change', this.selectDay);
-        this.time.addEventListener('change', this.selectTime);
+        this.day.addEventListener('change', this.selectRemindTime);
+        this.time.addEventListener('change', this.selectRemindTime);
         this.setBtn.addEventListener('click', this.setReminder);
     }
 
-    selectDay = (event) => {
+    selectRemindTime = (event) => {
```

Comment:
Try to rewrite this as I told you. Add event listener to `form` or `div`, on `change` or `input` event get the element, check it's value and set the button